src/server/pkg/worker/master.go

```diff
@@ -261,14 +265,20 @@ nextInput:
 
 // jobManager feeds datums to jobs
 func (a *APIServer) runJob(ctx context.Context, jobInfo *pps.JobInfo, pool *grpcutil.Pool) error {
+	jobStart := time.Now()
+	defer func() { protolion.Infof("running job (%v) took (%v)\n", jobInfo, time.Since(jobStart)) }()
+	protolion.Infof("running job (%v)\n", jobInfo)
 	pfsClient := a.pachClient.PfsAPIClient
 	objectClient := a.pachClient.ObjectAPIClient
 	ppsClient := a.pachClient.PpsAPIClient
 
 	jobID := jobInfo.Job.ID
 	var jobStopped bool
 	var jobStoppedMutex sync.Mutex
+	retryCount := 0
```

Comment:
Yup! But it's not present anywhere in the logs right? The goal here was to make it easier to read the logs and notice restart boundaries. This way I could even search the logs for a certain run of the datum.